internal/cli/root.go

```diff
@@ -41,26 +40,26 @@ func NewRootCommand(state *state.State, client hcapi2.Client) *cobra.Command {
 		DisableFlagsInUseLine: true,
 	}
```

Comment:
For example, we could load the config/client here using the `PersistentPreRun`.

Reply:
Not possible because fully configured state is needed for root command initialization -> Out of scope for this PR